.github/workflows/build.yaml

```diff
@@ -15,8 +15,8 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version: 1.20.1
+      - run: go install git.sr.ht/~charles/rq/cmd/rq@latest
+      - run: build/do.rq pull_request
```

Comment:
Just moving this up as I'd rather fail fast on builds than golangci-lint, which is... not fast.